src/pages/Login.vue

```diff
@@ -395,16 +397,32 @@ export default {
       // whether the phone number entered by the user is valid
       return this.phoneInput.toString().match(/^([0]|\+91)?[6-9]\d{9}$/g) != null;
     },
+    startCountdown(seconds) {
+      // counts seconds before enabling resend OTP button
+      this.counter = seconds;
+      const interval = setInterval(() => {
+          this.counter--;
+          if(this.counter == 0)
+          {
+            this.counting= false;
+            clearInterval(interval);
+          }

```

Comment:
once you create the computed property, won't have to set it manually.
```suggestion
          if (!this.counter) clearInterval(interval);
```

Reply:
I did this because I have to set counting to false.